pkg/auth/authenticator/password/keystonepassword/keystonepassword.go

```diff
@@ -0,0 +1,66 @@
+package keystonepassword
+
+import (
+	"fmt"
+	"errors"
+	"net/http"
+
+	"github.com/golang/glog"
+	"github.com/rackspace/gophercloud"
+	"github.com/rackspace/gophercloud/openstack"
```

Comment:
Ahh ok. These are already referenced in upstream kubernetes -- I wasn't sure if direct vs. indirect deps made a difference.